bats/README.md

```diff
@@ -0,0 +1,143 @@
+## Overview
+
+BATS is a testing framework for Bash shell scripts that provides supporting libraries and helpers for customizable server-side test automation.
```

Comment:
Not sure what "server-side" means here; Rancher Desktop is a desktop application.

```suggestion
BATS is a testing framework for Bash shell scripts that provides supporting libraries and helpers for customizable test automation.
```